charts/catalog/templates/rbac.yaml

```diff
@@ -86,7 +86,7 @@ items:
   # TODO: do not grant global access, limit to particular secrets referenced from servicebindings
   - apiGroups: [""]
     resources: ["secrets"]
-    verbs:     ["get","create","update","delete"]
+    verbs:     ["get","create","update","delete", "list", "watch"]
```

Comment:
client-go is listing secrets when the controller starts up now, i couldn't find a way to stop it from doing this.

Reply:
that's weird. probably some upstream change.

Reply:
we never have fixed that comment.